src/renderer/captureTool/js/capture.js

```diff
@@ -0,0 +1,1121 @@
+     *  - selectedPrefsSet
+     *  - selectedPrefsSetName
+     *  - prefsSetName
+     */
+    gpii.captureTool.watchKeyInOut = function (that, currentPage /*, isKeyedIn */) {
+        var transaction = that.applier.initiate();
+        transaction.fireChangeRequest({ path: "prefsSetSaveType", value: "existing"});
+        transaction.fireChangeRequest({ path: "selectedPrefsSet", value: "gpii-default"});
+        transaction.fireChangeRequest({ path: "selectedPrefsSetName", value: "GPII Default"});
+        transaction.fireChangeRequest({ path: "prefsSetName", value: "My Capture"});
+        transaction.commit();
+
+        if (currentPage === "4_save_choose_prefsset" ||
+            currentPage === "4_confirm_update_prefsset" ||
+            currentPage === "4_save_name")
+        {
+            that.applier.change("currentPage", "4_save_choose_prefsset");
+            that.render("4_save_choose_prefsset");
+        }
+    };
+
+    /**
+     * This function will take a schema from a supported setting in the solutions
+     * registry and return metadata on it's default value, including the type of
+     * the default value. The function will always return a block of json in order
+     * to avoid any ambiguities for a situation where the default value is perhaps
+     * `null`. A key `hasDefault` will be `true` or `false` to communicate this.
+     * If the schema does have a default value, then the properties `type` and
+     * `default` can also be included. Note that there may not always be a `type`
+     * that can be included, the inclusion of this in the return if optional and
+     * will be included if a `type` is declared.
+     *
+     * This is NOT meant to be an exhaustive coverage of all the possible ways
+     * default values can be set for any JSON Schema. We merely want to cover the
+     * cases in use in the solutions registry, focussing on single primitive values,
+     * and situations where we next one level deep due to a requirement of having a
+     * `properties` entry in the schema.
+     *
+     * @param {Object} schema - Schema to search for a default value. This is what
+     * would be in the solutions registry under `supportedSettings` and a
+     * configuration. An example from the Windows Mouse Keys MaxSpeed setting is:
+     * ```json
+     * {
+     *     "title": "Mouse keys speed",
+     *     "description": "Speed of mouse keys",
+     *     "type": "number",
+     *     "multipleOf": 10,
+     *     "default": 80
+     * }
+     * ```
+     * @return {Object} Always returns a json block. For the example above the
+     * return value would be:
+     * ```json
+     * {
+     *     "hasDefault": true,
+     *     "type": "number",
+     *     "default": 80
+     * }
+     * ```
+     * If a default value cannot be detected the return payload will be:
+     * ```json
+     * {
+     *     "hasDefault": false
+     * }
+     * ```
+     */
+    gpii.captureTool.findDefaultValue = function (schema) {
```

Comment:
I guess this is the code which duplicates what is in your universal pull. To start with, all the feedback given there is good for it here, (e.g. use of fluid.get) but further we seemed to determine in our meeting today with @the-t-in-rtf that it is probably not necessary

Reply:
Right... I mean, in theory it's not necessary, but in practice as long as the SPI settings are structured differently these user facing apps are still going to need to make these checks somewhere to format and check the default.  
Until the day when we try to no longer keep supported settings to nice single depth values, we won't need to the sort of recursive defaults checker we talked about on the meeting. But we will need a few special handlers for SPI settings, and maybe those RGB values.  I could simplify this method and remove some of the comments to make it smaller.

I did update it to use fluid.get